### PR TITLE
Fix vertical alignment of dropdown in navbar

### LINKED ISSUE #1607: Dark dropdown in navbar is not correctly vertically aligned
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Dark dropdown in navbar example in https://boosted.orange.com/docs/5.2/components/dropdowns/#dark-dropdowns is not correctly vertically aligned.

Screenshot 2022-11-15 at 07 43 19

I'm not sure but the height seems to be off as well compared to our global headers 🤷 

### Reduced test cases

See https://boosted.orange.com/docs/5.2/components/dropdowns/#dark-dropdowns.

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Firefox

### What version of Boosted are you using?

v5.2.1

### PR DESCRIPTION
**Pull request by MewenLeHo**
### Related issues

 Closes #1607 

### Description

Fix vertical alignment.

### Motivation & Context

Fix vertical alignment.

### Types of change

- Bug fix (non-breaking which fixes an issues)

### Live previews

https://deploy-preview-1625--boosted.netlify.app/docs/5.2/components/dropdowns/#dark-dropdowns

### Checklist

- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
- [x] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [x] My change introduces changes to the migration guide
- [x] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by julien-deramond** [comment]
Description must be updated to close automatically the issue with for example the keyword "Closes #issue-number"

**Comment by MewenLeHo** [comment]
Since it's a navbar example I kept the fix with the logo:
screenshot-deploy-preview-1625--boosted netlify app-2022 11 15-11_52_43

Do you prefer the fix without the logo?
screenshot-deploy-preview-1625--boosted netlify app-2022 11 15-11_53_10

File: site/content/docs/5.2/components/dropdowns.md [hunk 1]
```diff
@@ -230,7 +230,12 @@ And putting it to use in a navbar:
 {{< example >}}
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
   <div class="container-fluid">
-    <a class="navbar-brand" href="#">Navbar</a>
+    <div class="navbar-brand me-auto me-lg-4">
+      <a class="stretched-link" href="#">
+        <img src="/docs/5.2/assets/brand/orange-logo.svg" width="50" height="50" alt="Boosted - Back to Home" loading="lazy">
```
Comment (julien-deramond) [inline site/content/docs/5.2/components/dropdowns.md:235]: There is a discrepancy between what's inside Navbar examples and Global Headers examples regarding the `alt` information so I suggest to use this one (more like the navbar examples):

```suggestion
        <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" width="50" height="50" alt="Boosted" loading="lazy">
```

Anyway not in this PR we should align those `alt`s.

File: site/content/docs/5.2/components/dropdowns.md [hunk 2]
```diff
@@ -230,7 +230,12 @@ And putting it to use in a navbar:
 {{< example >}}
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
   <div class="container-fluid">
-    <a class="navbar-brand" href="#">Navbar</a>
+    <div class="navbar-brand me-auto me-lg-4">
```
Comment (julien-deramond) [inline site/content/docs/5.2/components/dropdowns.md:233]: In order to be consistent with the other pages, we shouldn't use here `me-auto me-lg-4` and rather make the modification everywhere in another PR.

```suggestion
    <div class="navbar-brand">
```

**Review by julien-deramond — CHANGES_REQUESTED** [review]
